src/themes/eui/eui_colors_dark.scss

```diff
@@ -1,25 +1,25 @@
 // Core
```

Comment:
Not sure you meant to commit this file (looks kinda cool, though)

Reply:
Whoops. From a video I was making on theming. Its solarized!